### PR TITLE
Various fixes to Makefile

### PR DESCRIPTION
**Pull request by ianpittwood**
- Fixed missing ":" in image tags
- Replaced "RSW_VERSION" with "RSW_TAG_VERSION" for image tags
- Changed reference in `test-all` from `rsp` to `rsw`

### CODE REVIEW

**Comment by ianpittwood** [comment]
@colearendt is the rsw-hooks command still in use? I tagged it with a TODO in the PR since workbench/hooks seems to have since been removed.

**Review by kfeinauer — APPROVED** [review]
LGTM

File: Makefile [hunk 1]
```diff
@@ -82,18 +82,18 @@ rsw: workbench
 workbench:  ## Build Workbench image
 	docker build -t rstudio/rstudio-workbench:$(RSW_TAG_VERSION) --build-arg R_VERSION=$(R_VERSION) --build-arg RSW_VERSION=$(RSW_VERSION) workbench
 
-rsw-hook:
+rsw-hook:  ## TODO: workbench/hooks is gone. Is this still needed?
```
Comment (colearendt) [inline Makefile:85]: Probably not!

**Review by colearendt — COMMENTED** [review]
Good catches! Feel free to remove `rsw-hook` if you want!

**Review by colearendt — APPROVED** [review]
LGTM!

**Pull request merged**